Marlin/src/inc/SanityCheck.h

```diff
@@ -998,9 +998,11 @@ static_assert(1 >= 0
 #endif
 
 #if HOTENDS > 1
-  #if TEMP_SENSOR_1 == 0
+  #if ENABLED(HEATER_1_USES_MAX6675) && !(defined(MAX6675_SS2) && MAX6675_SS2 >= 0)
```

Comment:
`HEATER_1_USES_MAX6675` doesn't appear to be defined anywhere. Changes are needed in `Conditionals_post.h`…
```cpp
#if TEMP_SENSOR_1 == -3
  #define HEATER_1_USES_MAX6675
  #define MAX6675_IS_MAX31855
  #define MAX6675_TMIN -270
  #define MAX6675_TMAX 1800
#elif TEMP_SENSOR_1 == -2
  #define HEATER_1_USES_MAX6675
  #define MAX6675_TMIN 0
  #define MAX6675_TMAX 1024
#elif TEMP_SENSOR_1 == -1
  #define HEATER_1_USES_AD595
#elif TEMP_SENSOR_1 == 0
  #undef HEATER_1_MINTEMP
  #undef HEATER_1_MAXTEMP
#elif TEMP_SENSOR_1 > 0
  #define THERMISTORHEATER_1 TEMP_SENSOR_1
  #define HEATER_1_USES_THERMISTOR
#endif
```